tests/perf-test/README.md

```diff
@@ -0,0 +1,69 @@
+# Performance testing
+
+We use Reassure for monitoring performance regression. It helps us check if our app is getting faster and quickly spot any issues we need to fix.
+
+## How does Reassure work?
+
+- Reassure builds on the existing React Testing Library setup and adds a performance measurement functionality. It's intended to be used on local machine and on a remote server as part of your continuous integration setup.
+- To make sure the results are reliable and consistent, Reassure runstests twice – once for the current branch and once for the base branch.
+
+## Performance Testing Strategy (`measurePerformance`)
+
+- The primary focus is on testing business cases rather than small, reusable parts that typically don't introduce regressions, although some tests in that area are still necessary
```

Comment:
```suggestion
- The primary focus is on testing business cases rather than small, reusable parts that typically don't introduce regressions, although some tests in that area are still necessary.
```